### PR TITLE
missing export of a constructor

### PR DESCRIPTION
**Pull request by kyessenov**
Signed-off-by: Kuat Yessenov <kuat@google.com>

### CODE REVIEW

File: attribute/stringmap.go [hunk 1]
```diff
@@ -30,10 +30,11 @@ type StringMap struct {
 }
 
 // NewStringMap instantiates a new string map.
-func NewStringMap(name string) StringMap {
+func NewStringMap(name string, entries map[string]string, owner Bag) StringMap {
```
Comment (ozevren) [inline attribute/stringmap.go]: This seems like a dangerous constructor as entries will be shared with the caller. I don't see the usage of copyValue, but it looks like it is a private method and it seems ok to keep the inline construction model in the original code (as opposed to exposing a constructor that can cause data-sharing issues).

Comment (kyessenov) [reply attribute/stringmap.go]: owner field is completely private. We don't have any other way to create a stringmap with an owner.

Comment (kyessenov) [reply attribute/stringmap.go]: I can add a comment saying that the stringmap should take ownership of the provided map.

Comment (ozevren) [reply attribute/stringmap.go]: how about just
```go
func NewMap(name string, owner Bag) StringMap {
}
```
then?

Comment (kyessenov) [reply attribute/stringmap.go]: This is not supposed to be a mutable map. How do we set both owner and entries in proto bag (which is not in this package)?

Comment (kyessenov) [reply attribute/stringmap.go]: Also, note that we already have `WrapStringMap` that accepts a map.

**Pull request merged**